Sources/XMTPiOS/Group.swift

```diff
@@ -82,6 +82,14 @@ public struct Group: Identifiable, Equatable, Hashable {
 			return []
 		}
 	}
+	
+	public var peerAddresses: [String] {
```

Comment:
when reading this, it wasn't clear to me ( initially ) that this is the "other conversation members peer addresses" list. not sure if it's a good idea or not, but the `memberAddresses.map(\.localizedLowercase)` seems to be error prone, as it might create different address representation to the same address. Ideally, it would be lower-cased at the source.

Reply:
That would be ideal but since we haven't been lowercasing from the beginning it's not that easy and would potentially break existing clients. Something we can strive for when we move to v3 tho.